eng/pipelines/templates/jobs/archetype-sdk-bom.yml

```diff
@@ -0,0 +1,78 @@
+parameters:
+  SDKType: bom
+  ServiceDirectory: not-specified # Set a default that breaks in obvious ways.
+  ValidationSteps: []
+
+jobs:
+  - job: 'Build and Validate'
+
+    variables:
+      - template: ../variables/globals.yml
+
+    pool:
+      vmImage: 'ubuntu-16.04'
+
+    strategy:
+      matrix:
+        Client Libraries:
+          ArtifactName: 'packages'
+          JavaVersion: '1.11'
+
+    steps:
+      - script: |
+          echo "##vso[build.addbuildtag]Scheduled"
+        displayName: 'Tag scheduled builds'
+        condition: and(succeeded(), eq(variables['Build.SourceBranchName'],'master'),eq(variables['Build.Reason'],'Schedule'))
+
+      - task: UsePythonVersion@0
+        displayName: 'Use Python 3.6'
+        inputs:
+          versionSpec: '3.6'
+
+      - template: tools/daily-dev-build-variable/daily-dev-build-variable.yml@azure-sdk-tools
+
+      - pwsh: |
+        $(Build.SourcesDirectory)/eng/versioning/pom_file_version_scanner.ps1
```

Comment:
Is this step actually necessary? The BOM file doesn't have tags, the right parentage to ensure that it's been scanned and even if it did, the script would have to be modified for this specific case to allow a dependencyManagement sections (which is effectively what a BOM file is).

Reply:
This can be removed, really the only checks I would want to do eagerly with the BOM is verifying that all dependencies references are in Maven central and that they align on common dependencies.

Reply:
Removed